roles/kibana/tasks/setup-Debian.yml

```diff
@@ -1,7 +0,0 @@
----
```

Comment:
Why is this being removed? Because it is untested?

Reply:
because as of know we agreed to only support Centos7 - and noted that in the README

Reply:
OK, sounds reasonable.